snapshots/SyncTest.json

```diff
@@ -1,3 +1,3 @@
 {
-  "getReserves": "3931"
+  "getReserves": "3973"
```

Comment:
Surprisingly this increase seems to be caused by `noIsolate` in `test_sync_multiple_unlocked`, even though this snapshot is from `test_settle_nonNative_withoutSync_loseFunds`. When I revert 82dda85feef02e8bdb23d83ca793d16807be8b8d it goes back to `3931`. I'm unsure why.

Reply:
This change also results in `3931`:
```
    function test_sync_multiple_unlocked() public /*noIsolate*/ {
        manager.sync(currency1);
        //assertEq(Currency.unwrap(currency1), Currency.unwrap(manager.getSyncedCurrency()));
        manager.sync(currency0);
        //assertEq(Currency.unwrap(currency0), Currency.unwrap(manager.getSyncedCurrency()));
    }
```

Reply:
I don't think it's a problem but I am curious.

Reply:
that is super odd... not sure

Reply:
i have been boggled by a few gas snapshot changes sometimes 🤷‍♂️

Reply:
imo this actually makes sense because of the way that `noIsolate` works! Its adding extra to the lastCallGas thats being snapshotted. So it isn't an actual increase to the v4 core code - its a testing-caused increase.

I could explain it more if you want to know more, but I'm not concerned nor surprised 🫡

Reply:
My surprise is because the affected snapshot happens in another test.

Reply:
I would like to understand if you have the time to explain

Reply:
> My surprise is because the affected snapshot happens in another test.
OH I did not see that 😆 ok now i no longer understand it 💀

Reply:
I agree that it has to do with `noIsolate` though